Mathlib/Algebra/Order/Field/Basic.lean

```diff
@@ -873,7 +878,10 @@ def evalInv : PositivityExt where eval {u α} zα pα e := do
 @[positivity _ ^ (0 : ℤ), Pow.pow _ (0 : ℤ)]
 def evalPowZeroInt : PositivityExt where eval {u α} _zα _pα e := do
   let .app (.app _ (a : Q($α))) _ ← withReducible (whnf e) | throwError "not ^"
-  _ ← synthInstanceQ (q(LinearOrderedSemifield $α) : Q(Type u))
+  let _a ← synthInstanceQ (q(Semifield $α) : Q(Type u))
+  let _a ← synthInstanceQ (q(LinearOrder $α) : Q(Type u))
+  let _a ← synthInstanceQ (q(IsStrictOrderedRing $α) : Q(Prop))
+  assumeInstancesCommute
```

Comment:
Just spotting things with are not "obviously" correct. I don't know anything about metaprogramming, so I don't know why you have inserted an `assumeInstancesCommute` here.